sqlconnect/internal/databricks/legacy_mappings.go

```diff
@@ -0,0 +1,48 @@
+package databricks
+
+import (
+	"strings"
+
+	"github.com/rudderlabs/sqlconnect-go/sqlconnect/internal/base"
+)
+
+func legacyColumnTypeMapper(columnType base.ColumnType) string {
+	columnTypeMappings := map[string]string{
+		"DECIMAL":       "int",
+		"NUMERIC":       "int",
```

Comment:
Decimal/Dec to int is a bug in legacy mapping? :(. Keeping a note to see if it can be fixed.